src/main/scala/ai/diffy/proxy/TimedMulticastService.scala

```diff
@@ -2,13 +2,37 @@ package ai.diffy.proxy
 
 import com.twitter.finagle.Service
 import com.twitter.util.{Future, Try}
+import com.twitter.finagle.http.Request
```

Comment:
Classes in ai.diffy.proxy are protocol agnostic. Any Http-specific features should be added in the HttpDifferenceProxy file.